widget/playground/src/hooks/useFontLoader.ts

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+
+import { getFontUrlByName } from '../utils/common';
+
+const useFontLoader = () => {
```

Comment:
This hook is only running in playground so it doesn't work on widget itself.

Steps to reproduce:
1. run `yarn dev:widget`
2. `widget/embedded/src/containers/Widget/Widget.tsx`: Change `<WidgetProvider config={...}>` to `<WidgetProvider config={{
    ...props.config,
    theme: {
      fontFamily: 'Barlow Condensed',
    },
  }}>`